### DISCUSSION TITLE
[Idea] Add a CONTRIBUTING.md

### DISCUSSION THREAD

**Discussion by burgerga** [community-scripts/ProxmoxVE#486] [Ideas]
First of all, thanks for taking this community effort up, very happy that this project continues, and als happy to get involved myself!

See https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors

This is THE standard for people to discover how they can get involved. It could link to the very handy https://github.com/community-scripts/ProxmoxVE/discussions/388 (@MickLesk) which I only discovered after preparing my pull request 😬 
And it should link to a guide/checklist with all script conventions that should be adhered to in preparing a new script, since not all existing scripts are completely up to the latest spec.

**Comment by MickLesk** [comment]
I've already prepared something like this, but haven't finalized it yet. We are currently still doing too much on the core (build.func / install.func) for a correct template.
